### ISSUE TITLE
Incorrect Published field

### ISSUE THREAD

**Issue by kalyparker** [MISP/MISP#6358]
Hello,

We have a weird issue on your instance. We get an event apparently having a publish date (get it during synchronization) but on the GUI we have this:
![image](https://user-images.githubusercontent.com/12010905/94430792-bea1e280-0194-11eb-9946-3c8af9cdfe71.png)
On my own instance, I have this:
![image](https://user-images.githubusercontent.com/12010905/94430980-0294e780-0195-11eb-86e8-95c34894c8f9.png)

The event is # 79839
Could you please have a look?

I clean this one on my side, but there is something weird here.
Cheers

**Comment by adulau** [comment]
Could you mention which version of MISP on each side (including commit id)? Thanks a lot.

**Comment by Rafiot** [comment]
That's the ID (79839) on our instance of misppriv.

**Comment by kalyparker** [comment]
Our side: v2.4.121 (ee0f708187dac85cab19115147c9d94becb55c05)
Misppriv, I do not have access to the precise commit id. Today, the footer is MISP 2.4.132

**Comment by mokaddem** [comment]
It could be that the sync process was interrupted at some point, thus preventing the event to be published.
Did you notice similar occurrence of this issue or could it be an isolated case?

**Comment by kalyparker** [comment]
No, this one was noticeable. I had a process which grab published events via pymisp and raise a lot of FP :(
Maybe you could spot it directly from the database?
